[Cloud multi-node] Add new approval steps (#1280)
timescale/docs: cloud/cloud-multi-node.md

### Setting up multi-node on Timescale Cloud

1.  [Log in to your Timescale Cloud account][cloud-login] and click `Create
    Service`.
1.  Go to the `Without demo data` tab to create the cluster without a data
    source. You can add your own data source later on. Click `Advanced options`
    to configure your service as a multi-node cluster.
1.  In the `Create a service` page, select the `Multi-node` service type.

> 1.  If this is your first multi-node service, a contact form appears. Multi-node

Reviewer: ```suggestion
1.  If this is your first multi-node service, you see a contact form. Multi-node
```

>     is currently available on request. A conversation with our customer support

Reviewer: ```suggestion
    is currently available on request. A conversation with the Timescale customer support
```

    team can help you understand the trade-offs and whether it's a good fit for
    your needs. Click `Contact us.`

> 1.  Our customer support team will contact you. Once your request is approved,

Reviewer: ```suggestion
1.  Our customer support team contacts you. When your request is approved,
```

    source. You can add your own data source later on. Click `Advanced options`
    to configure your service as a multi-node cluster.
1.  In the `Create a service` page, select the `Multi-node` service type.
1.  If this is your first multi-node service, you see a contact form. Multi-node
    is currently available on request. A conversation with the Timescale customer support
    team can help you understand the trade-offs and whether it's a good fit for
    your needs. Click `Contact us.`

> 1.  Our customer support team contacts you. When your request is approved,

Reviewer: ```suggestion
1.  The customer support team contacts you. When your request is approved,
```
